src/herbivoresAndCarnivores.js

```diff
@@ -1,19 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden = false) {
+    super(name, health);
+    this.hidden = hidden;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  constructor(name, health = 100, hidden) {
+    super(name, health, hidden);
+  }
+  bite(grass) {
```

Comment:
[CODE STYLE] The parameter name `grass` in the `bite` method of the Carnivore class should be more descriptive and indicate that it is expected to be an instance of Herbivore. A better name could be `herbivore` to clearly represent its expected type.